docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+nodejs_19    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_20    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_21    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_22    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_23    Event-driven I/O framework for the V8 JavaScript engine
+nodejs-10_x  Event-driven I/O framework for the V8 JavaScript engine
+
+Showing 10 of 67 results. Use `flox search nodejs --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+There is _a lot_ of software in the Flox catalog, so you should have no trouble finding the Node.js version you need. For this project, perhaps you initially decide to install Node.js v20.18.1, "Iron" (LTS).
+
+```zsh
+➜  node-project git:(main) ✗ flox install nodejs
+✅ 'nodejs' installed to environment 'node-project'
+```
+## Verify the Node.js version
+Now that you've installed Node.js, you activate the Flox environment to verify that it has the the version you expect.
+
+```zsh
+➜  node-project git:(main) ✗ flox activate
+✅ You are now using the environment 'node-project'.
+To stop using this environment, type 'exit'
+
+...
+flox [node-project] ➜  node-project git:(main) ✗ node -v
+v20.18.1
+```
+
+## Update the Node.js version
+If you want to install a different node version, you can edit your environment to include the version you need.
+
+Perhaps you decide to upgrade node version to v22. To achieve this, you run `flox uninstall nodejs`, followed by `flox install nodejs_22`.
+
+```zsh
+➜  node-project git:(main) ✗ flox uninstall nodejs
+🗑️  'nodejs' uninstalled from environment 'node-project'
+➜  node-project git:(main) ✗ flox install nodejs_22
+✅ 'nodejs_22' installed to environment 'node-project'
+➜  node-project git:(main) ✗ flox list
+nodejs_22: nodejs_22 (nodejs-22.10.0)
+➜  node-project git:(main) ✗ flox activate
+✅ You are now using the environment 'node-project'.
+To stop using this environment, type 'exit'
+
+...
+flox [node-project] ➜  node-project git:(main) ✗ node -v
+v22.10.0
+```
+
+If you'd like more granular control over the Node.js version to install, you can get the correct Flox catalog version name for the desired Node.js version by running `flox show <package>`:
+
+```zsh
+➜  node-project git:(main) ✗ flox show nodejs_22
+nodejs_22 - Event-driven I/O framework for the V8 JavaScript engine
+    nodejs_22@nodejs-22.10.0
+    nodejs_22@nodejs-22.9.0
+    nodejs_22@nodejs-22.8.0
+    nodejs_22@nodejs-22.6.0
+    nodejs_22@nodejs-22.5.1
+    nodejs_22@nodejs-22.4.1
+    nodejs_22@nodejs-22.3.0
+    nodejs_22@nodejs-22.2.0
+    nodejs_22@nodejs-22.0.0
+```
+
+Once you know the specific node 22 versions that are available, you can either run `flox install nodejs_22@nodejs-22.10.0`, or you can run `flox edit` to open up the manifest for the environment, as illustrated below. (For more on the manifest, read [our reference guide][manifest]{:target="\_blank"}.)
```

Comment:
> Once you know the specific node 22 versions that are available

I'm not sure what the naming convention is, but in this document there's Node.js, node, node 22, and node v22. We should probably pick one and be consistent.

Reply:
The discrepancy with versions (i.e., v22 vs just 22) was an oversight on my part, but using a mixture of Node.js and node was intentional. I think this is authentic to how a JavaScript developer would refer to the runtime. If the tone of this guide is meant to be formal, I'd replace everything instance with "Node.js," which is the branded term, I believe.